twim/calibration/R3BTwimMapped2Cal.h

```diff
@@ -88,13 +94,55 @@ class R3BTwimMapped2Cal : public FairTask
     std::vector<TArrayF*> CalEParams;
     std::vector<TArrayF*> PosParams;
 
-    Int_t mulanode[4][16 + 4];
-    Double_t fE[4][20][16 + 4];
-    Double_t fDT[4][20][16 + 4];
+  public:
+    class CalAnode
+    {
+      public:
+        CalAnode()
+            : fmult(0)
+        {
+            Init();
+        };
+        void Init();
+        void AddMult() { fmult++; };
+        void SetE(Double_t val) { fE.at(fmult) = val; };
+        void SetDt(Int_t val) { fDT.at(fmult) = val; };
+        void SetVal(Double_t ene, Int_t dtime)
+        {
+            fE.at(fmult) = ene;
+            fDT.at(fmult) = dtime;
+            fmult++;
+        };
+        auto GetMult() const { return fmult; };
+        auto GetE(Int_t mult) { return fE.at(mult); };
+        auto GetDT(Int_t mult) { return fDT.at(mult); };
+
+      private:
+        Int_t fmult;
```

Comment:
I remember that we had a consensus that c++ standard types (such as int, double, int8_t, etc) are preferred to ROOT types (Int_t, Double_t, etc). 

Please also don't forget to inline initialize those literal types.
```cpp
private:
    int fmult = 0;
```

Reply:
Oh sorry, I missed this comment. Following the suggestion by clang-tidy, I put a line to initialise the variable with
`Int_t fmult { 0 };`
Do you think I should modify all the ROOT types to C++ standard types?

Reply:
No, I don't think that's necessary. Just keep in mind that c++ standard types should be used when writing new code.